### ISSUE TITLE
Unable to connect on Mac (works on Ubuntu)

### ISSUE THREAD

**Issue by brandomr** [Joentje/nordvpn-proxy#63]
Running this on a Mac with `Docker Desktop 3.3.1`. I am stuck in this loop:

```
Tue Apr 27 16:12:08 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]185.247.70.115:443
Tue Apr 27 16:12:08 2021 Socket Buffers: R=[131072->131072] S=[16384->16384]
Tue Apr 27 16:12:08 2021 Attempting to establish TCP connection with [AF_INET]185.247.70.115:443 [nonblock]
Tue Apr 27 16:12:08 2021 TCP connection established with [AF_INET]185.247.70.115:443
Tue Apr 27 16:12:08 2021 TCP_CLIENT link local: (not bound)
Tue Apr 27 16:12:08 2021 TCP_CLIENT link remote: [AF_INET]185.247.70.115:443
Tue Apr 27 16:12:08 2021 Connection reset, restarting [0]
Tue Apr 27 16:12:08 2021 SIGUSR1[soft,connection-reset] received, process restarting
Tue Apr 27 16:12:08 2021 Restart pause, 5 second(s)
```

Note that as of a month or so ago this worked fine for me on a Mac. The exact same run command/credentials *do work* for me on Ubuntu running `Docker Engine - Community Version: 20.10.5`

**Comment by Joentje** [comment]
Strange that after a month it doesn't work anymore. Looks like a Openvpn issue. Maybe updating Openvpn could help.

**Comment by kbebee** [comment]
Curious if you got it corrected. I'm having the same issue.

**Comment by brandomr** [comment]
@scotthaleen keyed me into this being related to Mac Docker Desktop versions. I think if you update Docker Desktop it ought to work now, but haven't tried.

**Comment by kbebee** [comment]
I can't believe I'm about to say this..but that worked. My desktop app wasn't updating so I just redownloaded it and it worked. Thanks.

**Comment by scotthaleen** [comment]
There was definitely some issue with proxies reported on Docker "for-mac" version 3.3.1.  It seems they fixed them in 3.3.2+
